src/Components/Header/Header.css

```diff
@@ -0,0 +1,34 @@
+.main-navhead{
+    display: flex;
+    align-items: center;
+    height: 60px;
```

Comment:
Use relative units for responsiveness

Reply:
Will fix all issues in responsiveness after I am done with functionality